asterisk18.spec

```
@@ -0,0 +1,1979 @@
+BuildRequires:    unixODBC-devel
+%endif
+
+%if 0%{?postgresql}
+%if 0%{?rhel}
+BuildRequires:    postgresql-devel
+%else
+BuildRequires:    libpq-devel
+%endif
+%endif
+
+%if 0%{?radius}
+%if 0%{?fedora} || 0%{?rhel} < 7
+BuildRequires:    freeradius-client-devel
+%else
+BuildRequires:    radcli-compat-devel
+%endif
+%endif
+
+%if 0%{?snmp}
+BuildRequires:    net-snmp-devel
+BuildRequires:    lm_sensors-devel
+%endif
+
+%if 0%{?imap}
+BuildRequires:    uw-imap-devel
+%endif
+
+%if 0%{bundledjansson}
+BuildRequires:    jansson-devel
+Requires:         jansson
+%else
+Provides:         bundled(jansson) = 2.11
+%endif
+
+# for gpg to be able to verify the signature
+BuildRequires:    libgcrypt
+BuildRequires:    make
+
+Requires(pre):    %{_sbindir}/useradd
+Requires(pre):    %{_sbindir}/groupadd
+
+Requires(post):   systemd-units
+Requires(post):   systemd-sysv
+Requires(preun):  systemd-units
+Requires(postun): systemd-units
+
+
+# chan_phone headers no longer in kernel headers
+Obsoletes:        asterisk-phone < %{version}
+
+%description
+Asterisk is a complete PBX in software. It runs on Linux and provides
+all of the features you would expect from a PBX and more. Asterisk
+does voice over IP in three protocols, and can interoperate with
+almost all standards-based telephony equipment using relatively
+inexpensive hardware.
+
+%package core
+Summary: Asterisk core package without any "extras".
+Group: Utilities/System
+#Provides: %{name}-core
+#Provides: %{name}-speex
+Provides:  asterisk13-speex = 2:13.38.3-1
+Obsoletes: asterisk13-speex <= 2:13.38.3-2
+Provides:  asterisk13-resample = 2:13.38.3-1
+Obsoletes: asterisk13-resample <= 2:13.38.3-2
+Provides:  asterisk13-addons-core = 2:13.38.3-1
+Obsoletes: asterisk13-addons-core <= 2:13.38.3-2
+Provides:  asterisk13-ogg = 2:13.38.3-1
+Obsoletes: asterisk13-ogg <= 2:13.38.3-2
+Conflicts: asterisk14-core
+Conflicts: asterisk16-core
+Conflicts: asterisk10-core
+Conflicts: asterisk11-core
+Conflicts: asterisk12-core
+Provides:  asterisk13-core = 2:13.38.1-1
+Obsoletes: asterisk13-core <= 2:13.38.3-2
+Requires: openssl
+Requires: libxml2
+Requires: libsrtp23
+Requires(pre): %{_sbindir}/groupadd
+Requires(pre): %{_sbindir}/useradd
+
+%description core
+This package contains a base install of Asterisk without any "extras".
+
+%package addons
+Summary: Asterisk-addons package.
+Group: Utilities/System
+Requires: %{name}-addons-core = %{version}-%{release}
+Provides: %{name}-addons
+Provides: asterisk-13-addons
+
+%if 0%{?mysql}
+Requires: %{name}-addons-mysql = %{version}-%{release}
+Requires: mysql
+%endif
+
+%if 0%{?ooh323}
+Requires: %{name}-addons-ooh323 = %{version}-%{release}
+%endif
+
+%description addons
+This package contains a base install of Asterisk-addons without any "extras".
+
+%package addons-core
+Summary: Asterisk-addons core package.
+Group: Utilities/System
+Requires: %{name}-core = %{version}-%{release}
+Provides: %{name}-addons-core
+Provides: asteriak13-addons-core
```

Still, I'm not sure this is required.
```suggestion
Provides: asterisk13-addons-core
```